com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Motion-Blur.md

```diff
@@ -17,20 +17,25 @@ Motion Blur includes [more options](More-Options.md) that you must manually expo
+| **- Rotation Clamp**      | Use the slider to set the maximum velocity that HDRP allows Camera rotation to contribute to the velocities of GameObjects. This value is expressed in terms of screen fraction. Higher values result in Camera rotation giving wider blurs. This is only valid if **Camera Clamp Mode** is set to **Rotation** or **Separate Translation And Rotation**. |
+| **- Translation Clamp**   | Use the slider to set the maximum velocity that HDRP allows Camera translation to contribute to the velocities of GameObjects. This value is expressed in terms of screen fraction. Higher values result in Camera rotation giving wider blurs. This is only valid if **Camera Clamp Mode** is set to **Translation** or **Separate Translation And Rotation**. |
+| **- Motion Vector Clamp** | Use the slider to set the maximum velocity that HDRP allows Camera transform changes to contribute to the velocities of GameObjects. This value is expressed in terms of screen fraction. Higher values result in Camera rotation giving wider blurs. This is only valid if **Camera Clamp Mode** is set to **Full Camera Motion Vector**. |
 
 ## Details
 
 There are multiple options available to decrease the performance impact of Motion Blur. Listed in order of effectiveness, you can: 
 
 1. Reduce the **Sample Count**. A lower sample count directly translates to higher performance. However, it is important to keep in mind that the algorithm clamps the maximum amount of samples so that no two samples are less than a pixel apart, so a high sample count does not affect slowly moving GameObjects very much. 
 2. Increase the **Minimum Velocity**. Increase this threshold to make HDRP blur less of the screen. If many GameObjects are at a velocity below this threshold, HDRP does not calculate Motion Blur for them, and the resource intensity of the effect decreases.
-3. Decrease the **Maximum Velocity** and the **Camera Rotation Velocity Clamp**. This gives a less intense blur, which leads to an access pattern that is more friendly to the GPU. 
+3. Decrease the **Maximum Velocity** and the **Camera Clamp** parameters. This gives a less intense blur, which leads to an access pattern that is more friendly to the GPU. 
+
+It is important to consider that when selecting any **Camera Clamp Mode** different than **None**, motion vectors that are usually relative to camera motion will cease to be. As a simple example, if an object is parented to a Camera and it perfectly follow the camera, normally that object will have a motion vector with length close to 0, however if the camera is clamped differently than the object motion, the final velocity length will likely not be 0 if the clamping point is reached. 
```

Comment:
If you select a **Camera Clamp Mode** other than **None**, motion vectors that are usually relative to camera motion no longer are. For example, if an object has the camera as a parent and thus perfectly follows the camera, normally that object has a motion vector with a length close to zero. However, if the camera uses a different clamping method to object, the final velocity length is likely not zero if the clamping point is reached.